src/main/apollon-editor.ts

```diff
@@ -372,6 +372,8 @@ export class ApollonEditor {
         this.store.getState().lastAction.endsWith('DELETE')
       ) {
         const lastModel = ModelState.toModel(this.store.getState());
+        // eslint-disable-next-line no-console
+        console.log(lastModel);
```

Comment:
Can this be removed or is this log statement required for some sort of debugging?

Reply:
shoot that's for debugging, i'll remove it from the final code.